verifier/src/main/kotlin/tech/figure/classification/asset/verifier/provenance/AssetClassificationEvent.kt

```diff
@@ -57,17 +57,17 @@ class AssetClassificationEvent(
                             },
                             txHash = sourceTx.txResponse.txhash,
                             eventType = event.type,
-                            attributes = event.attributes.map { attribute ->
+                            attributes = event.attributesList.map { attribute ->
                                 Event(
-                                    key = attribute.key,
-                                    value = attribute.value
+                                    key = attribute.key.toStringUtf8(),
+                                    value = attribute.value.toStringUtf8()
```

Comment:
Sample of PM doing this conversion
https://github.com/FigureTechnologies/provenance-api-marketplace/blob/f9ed0fec77440574737f531033091660408c543a/portfolio-core/src/main/kotlin/io/provenance/portfolio/util/pb/PbExtensions.kt#L23